dag_in_context/src/schedule.egg

```diff
@@ -1,4 +1,5 @@
 (run-schedule
+  test-helpers
```

Comment:
nit: test-helpers is definitely a terrible name since it's not like the ruleset turns off for release

Reply:
suggestions?